lunatrace/bsl/hasura/migrations/lunatrace/1649717845749_create-users-table-use-as-auth-table/up.sql

```diff
@@ -21,9 +21,9 @@ ALTER TABLE public.organization_user ADD CONSTRAINT organization_user_user_id_fk
   REFERENCES public.users
   (id) ON UPDATE cascade ON DELETE cascade;
 
-UPDATE users SET github_id = (
+UPDATE public.users SET github_id = (
   SELECT kratos.github_id
   FROM (
-    SELECT config->'providers'->0->>'subject' as github_id, identity_id FROM identity_credentials JOIN identities ON identities.id = identity_credentials.identity_id
-  ) as kratos WHERE users.kratos_id = kratos.identity_id AND kratos.github_id IS NOT NULL
-)
+    SELECT config->'providers'->0->>'subject' as github_id, identity_id FROM public.identity_credentials JOIN public.identities ON public.identities.id = public.identity_credentials.identity_id
```

Comment:
Nice fixes, i just want to be careful this won't break anything in the prod db.  Because we have prod data now, we are having to make new migrations for all db changes.

Reply:
I think this is alright after discussing with Free and Chris, this shouldn't affect the existing schema. Prod has already run this migration so it will not execute it again and this does not change the result. We have a backup at the ready in case we need to restore.